lib/engine/g_1856/corporation.rb

```diff
@@ -49,13 +49,6 @@ def capitalization_type_desc
         CAPITALIZATION_STRS[@capitalization || _capitalization_type]
       end
 
-      def release_escrow!
```

Comment:
Intended change?

Reply:
Not really, but I'm fine with making this change considering this method never gets called

Reply:
I undid the change

Reply:
Sorry, I didn't mean for you to necessarily undo it, I just wanted to be sure you noticed it was included as it didn't seem related to the update. Sometimes I get distracted and change something and forget about it :D